superset/views/core.py

```diff
@@ -1195,6 +1195,20 @@ def copy_dash(self, dashboard_id):
                     old_id = "{}".format(value.get("meta").get("chartId"))
                     new_id = int(old_to_new_sliceids[old_id])
                     value["meta"]["chartId"] = new_id
+
+            if "filter_scopes" in data:
```

Comment:
This looks pretty similar to the chunk of code [here](https://github.com/apache/incubator-superset/blob/3efdfc53456aa702d01fee90c17b79ba9531c9b8/superset/models/dashboard.py#L341). Can we pull this out into a function?

Reply:
added a util function to clean up.